akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -86,14 +86,6 @@ private[persistence] trait Eventsourced
   private val writerUuid = UUID.randomUUID.toString
 
   private var journalBatch = Vector.empty[PersistentEnvelope]
-  // no longer used, but kept for binary compatibility
```

Comment:
lies

Reply:
Yeah, I can't see how a private field could affect bincomp?

Reply:
Example:
```
:javap -p akka.persistence.NamedPersistentActor

  private final int akka$persistence$Eventsourced$$maxMessageBatchSize;
  public int akka$persistence$Eventsourced$$maxMessageBatchSize();
  public final void akka$persistence$Eventsourced$_setter_$akka$persistence$Eventsourced$$maxMessageBatchSize_$eq(int);
```

Adding a field would not be allowed, removing might be ok, but do we dare to take the risk? Could we at least wait until 2.6?

Reply:
Alright, I'll revert that and see if I can silence it in some other way (I started out with but didn't get @unused to work)

Reply:
@silent should work

Reply:
reverted